Dockerfile

```
@@ -0,0 +1,17 @@
+# 第一阶段：使用 Python 镜像安装 Python 和需要的 Python 包
+FROM python:slim AS python-base
+RUN pip install uv
+
+# 第二阶段：使用 texlive 镜像，并安装 git，然后从第一阶段复制 Python
+FROM texlive/texlive:latest
```

Hadolint (2.12.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)